File: Course/ITP1/3_C/main.cpp
//
// AOJ ITP1 / 3_C - Swapping Two Numbers
//
// https://onlinejudge.u-aizu.ac.jp/courses/lesson/2/ITP1/all/ITP1_3_C

/* -------------------------------------------------------------------------- */

#include <bits/stdc++.h>
using namespace std;

#define rep(i, a, b) for (int i = (a); i < (b); i++)
using ll = long long;

int main()
{
    // [ input ]
    vector<pair<int,int>> xy;
    int n = 0;
    while (42) {
        // input
        int x, y;
        cin >> x >> y;
        if (!x && !y)
            break;
        xy.push_back({x, y});
        // update data count
        n++;
    }

    // [ output ]
    rep(i, 0, n) {
        // swap
        if (xy[i].first > xy[i].second)
            swap(xy[i].first, xy[i].second);
        // output
        printf("%d %d\n", xy[i].first, xy[i].second);
    }

    // [ return ]
    return (0);
}

File: Course/ITP1/3_C/main.py
#
# AOJ ITP1 / 3_C - Swapping Two Numbers
#
# https://onlinejudge.u-aizu.ac.jp/courses/lesson/2/ITP1/all/ITP1_3_C

# ---------------------------------------------------------------------------- #

def main():
    # [ input ]
    xy = []
    n = 0
    while 42:
        # input
        x, y = map(int, input().split())
        if x == 0 and y == 0:
            break
        xy.append([x, y])
        # update data count
        n += 1

    # [ output ]
    for i in range(n):
        # swap
        if xy[i][0] > xy[i][1]:
            xy[i][0], xy[i][1] = xy[i][1], xy[i][0]
        # output
        print("%d %d" % (xy[i][0], xy[i][1]))

    # [ return ]
    return

# ---------------------------------------------------------------------------- #

if __name__ == '__main__':
    main()
